specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -45,6 +45,15 @@ tag: package-2019-06-preview
 ```
 
 
+
+### Tag: package-2021-04
+
+These settings apply only when `--tag=package-2021-04` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-04'
+input-file:
+  - Microsoft.AlertsManagement/stable/2021-04-01/SmartDetectorAlertRulesApi.json
```

Comment:
I suppose this tag cannot cover the full functionalities of this RP, could you please append a `-only` to the name of this tag to emphasize?

Reply:
done.
also fixed the linter issues